packages/visual-editing/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sanity/visual-editing",
-  "version": "1.8.20",
+  "version": "1.8.21-canary.0",
```

Comment:
Can you explain this change to me?

Reply:
I just ran a canary release locally so I could compare the output on esm.sh as used in the PR description :)

Reply:
I didn't mean to commit that though hang on

Reply:
Fixed